src/ConsentRequiredController.js

```diff
@@ -0,0 +1,135 @@
+        }).then(function () {
+          Util.redirect(self.settings.get('consentCancelLink'));
+        });
+      }
+    },
+    Form: {
+      noButtonBar: true,
+      formChildren: function () {
+        var transaction = this.options.appState.get('transaction');
+        return [
+          FormType.View({
+            View: new ConsentHeader({
+              orgLogo: this.settings.get('logo'),
+              userFirstName: transaction.user.profile.firstName,
+              userLastName: transaction.user.profile.lastName
+            })
+          }),
+          FormType.View({
+            View: new ConsentBeacon({
+              clientLogo: transaction.target.logo && transaction.target.logo.href,
+            })
+          }),
+          FormType.View({
+            View: Okta.View.extend({
+              className: 'consent-title',
+              template: '\
+                <p><b>{{appName}}</b> {{i18n code="consent.required.title" bundle="login"}}</p>\
+              ',
+              getTemplateData: function () {
+                return { appName: transaction.target.label };
+              }
+            })
+          }),
+          FormType.View({
+            View: new ScopeList({ model: this.model })
+          }),
+          FormType.View({
+            View: Okta.View.extend({
+              className: 'consent-description',
+              template: '\
+                <p>{{i18n code="consent.required.description" bundle="login"}}</p>\
+                {{#if termsOfService}}\
+                  <a class="terms-of-service" href="{{termsOfService}}" target="_blank">\
+                    {{i18n code="consent.required.termsOfService" bundle="login"}}\
+                  </a>\
+                  {{#if privacyPolicy}}\
+                    &#8226\
+                  {{/if}}\
+                {{/if}}\
+                {{#if privacyPolicy}}\
+                  <a class="privacy-policy" href="{{privacyPolicy}}" target="_blank">\
+                    {{i18n code="consent.required.privacyPolicy" bundle="login"}}\
+                  </a>\
+                {{/if}}\
+              ',
+              getTemplateData: function () {
+                var termOfServiceUrl;
+                var privacyPolicyUrl;
+                if (transaction.target['terms-of-service']) {
+                  termOfServiceUrl = transaction.target['terms-of-service'].href;
+                }
+                if (transaction.target['privacy-policy']) {
+                  privacyPolicyUrl = transaction.target['privacy-policy'].href;
+                }
+                return {
+                  termsOfService: termOfServiceUrl,
+                  privacyPolicy: privacyPolicyUrl
+                };
+              }
+            })
+          }),
+          FormType.View({
+            View: new ConsentButtonsBar({ model: this.model })
+          })
+        ];
+      },
+      preRender: function () {
+        $('.okta-sign-in-header').hide();
```

Comment:
I'm sure UX thought about this, but out of curiosity, why can't we reuse the header we already have?

Reply:
this UX is wrong and a temporary solution, UX didn't base the design on actual signin widget design and is working on a new one, this will change completely so for now we decided to go with this solution